install_config/registry/accessing_registry.adoc

```diff
@@ -272,3 +272,42 @@ cf2616975b4a: Image successfully pushed
 Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 ====
+
+[[accessing-metrics]]
+== Accessing Metrics
+
+The registry provides an endpoint for
+link:https://prometheus.io/docs/introduction/overview/[Prometheus metrics].
+Prometheus is a standalone open-source systems monitoring and alerting toolkit.
+
+The metrics are exposed at `/extensions/v2/metrics` path of the registry endpoint.
+This route needs to be
+xref:../registry/extended_registry_configuration.adoc#docker-registry-configuration-reference-openshift[enabled
+first] though.
```

Comment:
However, this route needs to be xref:../registry/extended_registry_configuration.adoc#docker-registry-configuration-reference-openshift[enabled] first.